.ci_support/migrations/cudnn9.yaml

```diff
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 2
```

Comment:
Found out why, the build_number in this migrator was set to `1` previously at commit 259805b7fdfe097b473773329a341a155fd61dd2. I've bumped it to `2` at commit 61d90a651b2c37237041ae9fb3e4e71d214e7c14 and re-rerendered.

Reply:
@hmaarrfk - do you have a clue what's going on here?

Reply:
I think it was just a copy-paste error. @hmaarrfk copied the `cudnn9.yaml` file from somewhere that set `build_number: 1`, when it should have been `build_number: 2`.

Reply:
I think `build_number: 1` is how it's done usually – it's an increment in the `build_number`.

Reply:
> I think `build_number: 1` is how it's done usually – it's an increment in the `build_number`.

Hmm ok. I did a quick search and found https://github.com/regro/cf-scripts/blob/621910b1438b5ed2f3510fc72c7f692ac4e2ed5a/README.md?plain=1#L99-L103, which has a comment that says `build_number` determines the increment, but then the actual uncommented line says `bump_number: 1` :confused: